server/routes/api.js

```diff
@@ -57,6 +51,33 @@ var connection = mysql.createPool({
   password: process.env.password,
   database: process.env.database,
 });
+router.post("/uploadProfileImage/:id/:userType", upload.single('updateImageInput'), (req, res) => {
+  if (!req.file) {
+    return res.send({
+      success: false
+    });
+  } 
+
+  const data = readImageFile('src/assets/user-profile-images/' + req.file.filename);
```

Comment:
I am not sure this will work on a server.
Because on a server we have different routes.
For example, we don't have src in the root.
Maybe we can storage in same place like we have for documents in server folder just create new folder(uploads/user-profile-image) cb(null, "./server/routes/uploads/");
Or better way will be if we can dynamically read the root path and concatenate then 'assets/user-profile-images'.